templates/company/view.html.tx

```diff
@@ -375,10 +406,17 @@
             </table>
     %    }
     % }
-    % if $company.foreign_company_details {
+    % if $company.foreign_company_details || $company.type {
         <h2 class="heading-medium">Company details in the country of incorporation</h2>
        % my $foreign_details = $company.foreign_company_details;
+       % my $company_type = $company.type;
         <div class="grid-row">
+        % if $company_type {
```

Comment:
I think this should be `% if $company.type == "registered-overseas-entity" {` also